framework/src/loops/UpdateDisplacedMeshThread.C

```diff
@@ -142,6 +143,12 @@ UpdateDisplacedMeshThread::onNode(NodeRange::const_iterator & nd)
   }
 }
 
+void
+UpdateDisplacedMeshThread::post()
+{
+  _displaced_problem.mesh().getMesh().clear_point_locator();
```

Comment:
This seems obviously correct ... which makes it somewhat concerning that it wasn't being done years ago.  Did we never run into a problem until we tried to do a transfer to/from a displaced mesh that relied on a PointLocator query?

Reply:
It is frankly shocking that no one complained about this before

Reply:
many kudos to @maxnezdyur for finding it and reporting it